### PR TITLE
Bing geocode

### PR DESCRIPTION
**Pull request by pcw11211**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/polling/functions.inc.php [hunk 1]
```diff
@@ -575,6 +584,8 @@ function location_to_latlng($device)
             curl_setopt($curl_init, CURLOPT_CONNECTTIMEOUT, 5);
             $data = json_decode(curl_exec($curl_init), true);
             // Parse the data from the specific Geocode services.
+           d_echo($api_url."\n");
```
Comment (laf) [inline includes/polling/functions.inc.php]: Can you remove these two lines. We'd expose people's API keys here and a lot of users will just upload the output of debug without sanitsing the data.

**Comment by pcw11211** [comment]
fixed exposed api keys
left print_r($data) for easier troubleshooting
changed geocoding with google without key and set msg "No geocode API key set\n"

File: includes/polling/functions.inc.php (hunk 1 above)
Comment (pcw11211) [reply includes/polling/functions.inc.php]: fixed exposed api keys
left print_r($data) for easier troubleshooting
changed geocoding with google without key and set msg "No geocode API key set\n"

**Comment by laf** [comment]
Thanks very much for contributing :)

**Pull request merged**